assets/scss/components/_paginator.scss

```diff
@@ -0,0 +1,37 @@
+.c-paginator {
```

Comment:
@mgbelegu I suggest renaming this component to `c-pagination`, which describes the section. The term `c-paginator` implies its function rather than being a section name.